ui/apos/components/AposExportModal.vue

```diff
@@ -452,15 +433,18 @@ export default {
   width: 100%;
 }
 
-.apos-export__section-container {
-  height: var(--container-height);
-  transition: height 200ms linear;
-}
+.apos-export__section--related-documents {
+  overflow: hidden;
+  height: 315px;
```

Comment:
we have to set a height to make the transition work.
this value respects what we currently have

Reply:
i think you just need `max-height` to get your transition